src/coreclr/jit/codegencommon.cpp

```diff
@@ -2097,6 +2097,26 @@ void CodeGen::genEmitUnwindDebugGCandEH()
     getDisAssembler().disAsmCode((BYTE*)*codePtr, finalHotCodeSize, (BYTE*)coldCodePtr, finalColdCodeSize);
 #endif // LATE_DISASM
 
+#ifdef DEBUG
+    if ((compiler->opts.disAsm || compiler->verbose) && compiler->opts.disAsmHexDumpFile)
```

Comment:
Why do you need to set disAsm or verbose? Couldn't this work fine with just setting the new variable?

Reply:
+1 on that.

Reply:
There could be:
```
CONFIG_METHODSET(JitRawHexCode,...
CONFIG_STRING(JitRawHexCodeFile,...
```
used as:
```
DOTNET_JitRawHexCode=foo bar baz
DOTNET_JitRawHexCodeFile=out.asm
```

Reply:
> although, one of the reason could be that we just want to output bytes for a particular method instead of all the methods.

That was the idea, but it would be fine to also change it to have the separate variables like @BruceForstall mentioned. I can do that.

Reply:
@BruceForstall @kunalspathak Wouldn't we need `verbose` if we want to capture the code when running the emitter tests?

Reply:
`genArm64EmitterUnitTests` currently only runs under `verbose`, but that's a separate issue, and can be changed (especially given this functionality)